tsdb/engine/tsm1/wal_test.go

```diff
@@ -701,6 +703,124 @@ func TestWALRollSegment(t *testing.T) {
 	}
 }
 
+func TestWAL_DistSize(t *testing.T) {
+	test := func(w *tsm1.WAL, oldZero, curZero bool) () {
+		// get disk size by reading file
+		files, err := ioutil.ReadDir(w.Path())
+		if err != nil {
+			t.Fatalf("read dir err: %v", err)
+		}
+
+		sort.Slice(files, func(i, j int) bool {
+			return files[i].Name() < files[j].Name()
+		})
+
+		var old, cur int64
+		for i, f := range files {
```

Comment:
This loop confuses me. Is the end goal to have:
```
cur = // size of last file
old = // sum of sizes of all but last file
```
?

Reply:
If so, would we assign `cur` directly outside of the loop, and have the `range` only loop over the files we want to sum?